$MirOS: ports/print/cups/patches/patch-cups_Makefile,v 1.3 2006/07/07 16:31:03 bsiegert Exp $
--- cups/Makefile.orig	Thu Aug  3 04:20:34 2006
+++ cups/Makefile	Wed Nov  8 18:01:54 2006
@@ -163,9 +163,9 @@ depend:
 install:	all installhdrs $(INSTALLSTATIC) $(INSTALL32) $(INSTALL64)
 	$(INSTALL_DIR) -m 755 $(LIBDIR)
 	$(INSTALL_LIB) $(LIBCUPS) $(LIBDIR)
-	if test $(LIBCUPS) = "libcups.so.2" -o $(LIBCUPS) = "libcups.sl.2"; then \
-		$(RM) $(LIBDIR)/`basename $(LIBCUPS) .2`; \
-		$(LN) $(LIBCUPS) $(LIBDIR)/`basename $(LIBCUPS) .2`; \
+	if test $(LIBCUPS) = "libcups.so.2.2" -o $(LIBCUPS) = "libcups.sl.2.2"; then \
+		$(RM) $(LIBDIR)/`basename $(LIBCUPS) .2.2`; \
+		$(LN) $(LIBCUPS) $(LIBDIR)/`basename $(LIBCUPS) .2.2`; \
 	fi
 	if test $(LIBCUPS) = "libcups.2.dylib"; then \
 		$(STRIP) -x $(LIBDIR)/$(LIBCUPS); \
@@ -292,7 +292,7 @@ libcups_s.a:	$(LIBOBJS) libcups_s.exp
 
 libcups.la:    $(LIBOBJS)
 	echo Linking $@...
-	$(CC) $(ARCHFLAGS) $(DSOFLAGS) -o $@ $(LIBOBJS:.o=.lo) -rpath $(LIBDIR) \
+	$(DSO) $(ARCHFLAGS) $(DSOFLAGS) -o $@ $(LIBOBJS:.o=.lo) -rpath $(LIBDIR) \
 		-version-info 2:7 $(SSLLIBS) $(COMMONLIBS) $(LIBZ)
 
 
